composer.json

```diff
@@ -29,7 +29,7 @@
     "minimum-stability": "dev",
 	"prefer-stable": true,
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=7.3.0",
```

Comment:
Do we need to change the php version?

Reply:
I changed it because the phpunit version used in composer.json is 9.5+ and it requires php 7.3+ (Ref: https://github.com/sebastianbergmann/phpunit/blob/9.5/composer.json#L24)

If the change is to big for the version 11.x we may add this to the next major version. 

Another way to solve this would be to downgrade phpunit to version 8.x but support for it ends in a couple of days (February 5 2023). Or leave the php version unchanged.

Reply:
Thank you. We should be okay with 7.3